base/src/main/java/uk/ac/ebi/atlas/solr/cloud/admin/SolrCloudAdminProxy.java

```diff
@@ -0,0 +1,81 @@
+package uk.ac.ebi.atlas.solr.cloud.admin;
+
+import org.apache.commons.collections.MapUtils;
+import org.apache.solr.client.solrj.SolrRequest;
+import org.apache.solr.client.solrj.SolrServerException;
+import org.apache.solr.client.solrj.impl.CloudSolrClient;
+import org.apache.solr.client.solrj.request.CollectionAdminRequest;
+import org.apache.solr.common.util.NamedList;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+@Component
+public class SolrCloudAdminProxy {
+    private final CloudSolrClient cloudSolrClient;
+
+    public SolrCloudAdminProxy(@Qualifier("zkHost") String zkHost) {
+        cloudSolrClient = new CloudSolrClient.Builder().withZkHost(zkHost).build();
+    }
+
+    public boolean areCollectionsUp(List<String> collectionNames, String... aliasedCollectionNames) throws IOException, SolrServerException {
+        List<String> aliases = Arrays.asList(aliasedCollectionNames);
+        SolrRequest request = new CollectionAdminRequest.ClusterStatus();
+
+        ArrayList<String> allCollectionNames = new ArrayList<>(collectionNames);
+
+        NamedList<Object> response = cloudSolrClient.request(request);
+
+        // Get real collection names for each alias
+        aliases.forEach(alias -> allCollectionNames.add(getCollectionNameForAlias(response, alias)));
+
+        List<String> statuses = allCollectionNames
+                .stream()
```

Comment:
Instead of the list-wrangling above, I think the following reads better:
```
List<String> statuses = 
        Stream.concat(
            collectionNames,
            Arrays.stream(aliasedCollectionnames).map(alias -> getCollectionNameForAlias(response, alias))
        .flatMap...
```